src/RedisInterface/Services/DashboardService.cs

```diff
@@ -176,21 +177,9 @@ public async Task<List<actionSequenceResponse>> GetActionSequenceAsync()
         /// <returns></returns>
         public async Task<List<string>> GetOnboardingItemNamesAsync()
         {
-            List<string> tempOnboardItemTypes = new List<string>();
-            var cloud = new CloudModel();
-            var edge = new EdgeModel();
-            var robot = new RobotModel();
-            var instance = new InstanceModel();
-            var action = new Middleware.Common.Models.ActionModel();
-            var task = new Middleware.Common.Models.TaskModel();
-            tempOnboardItemTypes.Add(cloud.GetType().ToString());
-            tempOnboardItemTypes.Add(edge.GetType().ToString());
-            tempOnboardItemTypes.Add(robot.GetType().ToString());
-            tempOnboardItemTypes.Add(instance.GetType().ToString());
-            tempOnboardItemTypes.Add(action.GetType().ToString());
-            tempOnboardItemTypes.Add(task.GetType().ToString());
-
-            return tempOnboardItemTypes;
+            var types = new Type[] { typeof(CloudModel), typeof(EdgeModel), typeof(RobotModel), typeof(InstanceModel) };
```

Comment:
One question, previously you also had an ActionModel and TaskModel in here, are they not needed?